Api/NodeTypeInterface.php

```diff
@@ -29,7 +29,7 @@ public function fetchData(array $nodes);
      * @param int $level in tree depth
      * @return string
      */
-    public function getHtml($nodeId, $level);
+    public function getHtml(int $nodeId, int $level);
```

Comment:
This change requires increase php dependency in `composer.json`

Reply:
@DariuszMaciejewski Could you confirm?

Reply:
Yeah, but we don't want to do that yet (if it isn't necessary).

Reply:
removed